data/hai/hai reveal 5 philosophers.txt

```diff
@@ -212,22 +212,72 @@ mission "Hai Reveal [B03] Contact Giti"
 		has "Hai Reveal [B02] Join the Fleet: done"
 	on offer
 		conversation
-			`While you join with the fleet that lands to occupy the spaceport a number of those involved in the assault descend to other areas of the planet, pursuing priority targets who revealed a little too much about their position in the attempted defense of <origin>. This leads to a significant wait until those are resolved and the port is secure before everyone needed can reconvene to discuss the next steps.`
-			`	You eventually end up in the secured command area hastily set up in the spaceport as Choot'k and Danforth come in discussing the tactical minutiae of some location you're unfamiliar with. They are trailed by Batu, who looks to be doing his best to pay close attention, and eventually joined by Sayari and Teeneep. One of Danforth's lieutenants brings up the rear and closes the door behind you all.`
-			`	"So," begins Danforth, "Stage one has been successful. However, it looks like we're going to be here a while. Stormhold is an absolute rabbit warren of hidden places. Even with the Hai's ships and resources on top of everything we've brought, it could take several days, or even weeks, to be satisfied that we've rooted things out sufficiently."`
-			`	Batu clears his throat and Danforth glances at him before evidently remembering something discussed earlier. "I'm sorry Elder Teeneep. I must advise you that at this stage all hope of keeping the Hai a secret is probably lost now. Even if we could somehow convince all the factions here to keep the secret, that blockade has drawn enough attention that really all that remains is to try and make sure issues are resolved before it becomes irretrievably common knowledge.`
-			`	"For now, all communication in and out of the area is blocked at the relay unless its to or from specific high-level Navy or Syndicate sources. Captain <last>, I'm afraid that also includes you. No communications in or out since we don't have the time or equipment to properly harden your systems against potential intrusion, but you do have the nearly exclusive freedom to leave and re-enter through the blockade. You're a pilot, not a ground combat troop operator, so I suspect that since we'll be here a while there'll be other things for you to do."`
+			`While you join the fleet that lands to occupy the spaceport, other assault ships pursue priority targets elsewhere on <origin>. This leads to a significant wait until the port is secure before everyone needed can reconvene to discuss the next steps.`
```

Comment:
```suggestion
			`While you join the fleet that lands to occupy the spaceport, other assault ships pursue priority targets elsewhere on <origin>. There's a significant wait to ensure that the port is secure before everyone needed can reconvene to discuss the next steps.`
```
Surely there's enough ships around to occupy Stormhold's main spaceport? There's probably more ships here in Alcyone than fought in the entire Free Worlds war.

Reply:
The point being made here is that Stormhold is, indeed, a planet of fortresses, as the planet description implies. There are numerous pirates on the ground in well-fortified positions, shooting at anyone who enters the spaceport, so you have to wait for special ops troops to secure the area. Subduing a well-armed and hostile population of millions holed up in bunkers will take wave after wave of troop transports and a prolonged operation. Even one spaceport is a major operation.

Reply:
Alright, but I still think that my suggestion sounds better in general.

Reply:
Yes, sorry, I misread it. I see what's bothering me now: it should be "port has been secured" rather than "port is secure." I will do that, and put in your changes too.

Reply:
I rewrote that part and (I think) I incorporated your suggestions.